master/internal/kubernetes/pod.go

```diff
@@ -117,6 +122,11 @@ func (p *pod) Receive(ctx *actor.Context) error {
 			return err
```

Comment:
question: if a pod actor is created, it must wait until the k8s pod to be created. Right?

Reply:
Yeah it creates the spec (but not the actual pod) before getting to the main loop. The actual creation now happens in the main loop once the token is received.

Reply:
Yeah definitely, updated these names.